tests/codeception/acceptance/102-BackendDeveloper/BackendDeveloperCest.php

```diff
@@ -99,7 +99,7 @@ public function fileManagementViewEditTemplatesTest(\AcceptanceTester $I)
         $I->see('css/',         Locator::href("/bolt/files/theme/$dir/css"));
         $I->see('images/',      Locator::href("/bolt/files/theme/$dir/images"));
         $I->see('javascripts/', Locator::href("/bolt/files/theme/$dir/javascripts"));
-        $I->see('config.yml',   Locator::href("/bolt/file/edit/theme/$dir/config.yml"));
+        $I->see('theme.yml',    Locator::href("/bolt/file/edit/theme/$dir/theme.yml"));
```

Comment:
Ha! Not @bobdenotter but @Nitpick-CI has had a few :beers: (just joking, on all points)